sci-electronics/systemc/systemc-2.3.4-r2.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PV="${PV}_pub_rev_20190614"
+
+inherit autotools toolchain-funcs
+
+DESCRIPTION="A C++ based modeling platform for VLSI and system-level co-design"
+HOMEPAGE="
+	https://systemc.org
+	https://github.com/accellera-official/systemc
+"
+
+if [[ "${PV}" == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/accellera-official/${PN}.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/accellera-official/${PN}/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~riscv ~x86"
+	S="${WORKDIR}/${PN}-${MY_PV}"
+fi
+
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE="debug doc examples static-libs test"
```

Comment:
The test USE flag does not appear to be used anywhere, I think we can drop it.

Reply:
Done, dropped unused test USE flag.